config/checkstyle-input-suppressions.xml

```diff
@@ -68,46 +68,6 @@
           files="[\\/]packageobjectfactory[\\/]zoo[\\/]FooCheck.java"/>
 
   <!-- until https://github.com/checkstyle/checkstyle/issues/13213 -->
-  <suppress id="UnnecessaryOkComment"
```

Comment:
Checkstyle requires all your changes in a single commit : 
https://circleci.com/gh/checkstyle/checkstyle/524209

Squash and rebase if necessary.

Reply:
I am facing trouble in squashing merge commits. Please guide me to the right direction.

Reply:
@mayankch283 ,

Checkstyle is very well documented and contributor-friendly.
I suggest you go through the following page for clarifications : https://checkstyle.org/beginning_development.html